runtime/tests/tests.rs

```diff
@@ -242,37 +366,31 @@ fn staking_validators_should_receive_equal_transaction_fee_reward() {
 
 			let fm = TransactionPayment::next_fee_multiplier();
 			let fee = transfer_fee(&xt, fm, &runtime_call);
-			let fee_reward = fee / validators.len() as u128;
-			let remainder = fee % validators.len() as u128;
+			let per_fee_reward = fee / validators.len() as Balance;
 
-			let previous_total_issuance = GenericAsset::total_issuance(&CENTRAPAY_ASSET_ID);
```

Comment:
Optional:
I think we should check if the total issuance is adjusted correctly in this test
I.e. total issuance in the new era = previous_issuance + total_staking_reward  - transaction_fee_remainder - staking_reward_remainder

Reply:
Thanks for pointing this out. This will be addressed